express-zod-api/src/documentation-helpers.ts

```diff
@@ -289,253 +253,81 @@ export const depictDateOut: Depicter = ({}: DateOutSchema, ctx) => {
   };
 };
 
-/** @throws DocumentationError */
-export const depictDate: Depicter = ({}: z.ZodDate, ctx) => {
-  throw new DocumentationError(
-    `Using z.date() within ${
-      ctx.isResponse ? "output" : "input"
-    } schema is forbidden. Please use ez.date${
-      ctx.isResponse ? "Out" : "In"
-    }() instead. Check out the documentation for details.`,
-    ctx,
-  );
-};
-
-export const depictBoolean: Depicter = () => ({ type: "boolean" });
-
-export const depictBigInt: Depicter = () => ({
-  type: "integer",
+export const onBigInt: Depicter = () => ({
```

Comment:
maybe should restore their original names since already managed to restore their immutability